qemu/tests/blockdev_mirror_to_rbd.py

```diff
@@ -0,0 +1,36 @@
+from provider.blockdev_mirror_wait import BlockdevMirrorWaitTest
+from provider.virt_storage.storage_admin import sp_admin
+
+
+class BlockdevMirrorRBDNode(BlockdevMirrorWaitTest):
+    """
+    Block mirror with filter-node-name set test
```

Comment:
wrong comment
also add 'pass' to the class definition

Reply:
Have updated, thanks zhenchao